charts/shoot-core/components/charts/network-policies/templates/allow-calico-to-api-server.yaml

```diff
@@ -0,0 +1,20 @@
+kind: NetworkPolicy
```

Comment:
Can you rename the file to `allow-to-api-server.yaml`?

Reply:
thanks, i've renamed the file